backend/internal/models/media.go

```diff
@@ -17,8 +17,14 @@ type Media interface {
 	GetMediaType() MediaType
 }
 
+// names dont get much better than this
+type MediaWithReviewCountAndType struct {
+	Media Media     `json:"media"`
+	Count int       `json:"review_count"`
+	Type  MediaType `json:"type"`
+}
```

Comment:
ooh good question - i agree with u, i think out of the 3 places you suggested this is the best place even though it's specific to just the one handler. maybe a new package altogether could be another option so that models stays clean and represents only our "pure" types, but tbh the way you have it is fine with me